R/redist-smc.R

```diff
@@ -133,33 +145,33 @@
 #'                             ))
 #' }
 #'
+#' @concept simulate
 #' @md
-#' @importFrom stats qnorm
 #' @export
 redist.smc = function(adj, total_pop, nsims, ndists, counties=NULL,
-                      pop_tol = 0.01, compactness=1,
+                      pop_tol = 0.01, pop_bounds=NULL, compactness=1,
                       constraints=list(),
                       resample=TRUE,
                       constraint_fn=function(m) rep(0, ncol(m)),
-                      adapt_k_thresh=0.95, seq_alpha=0.1+0.2*compactness,
+                      adapt_k_thresh=0.975, seq_alpha=0.2+0.2*compactness,
                       truncate=(compactness != 1),
                       trunc_fn=function(x) pmin(x, 0.01*nsims^0.4),
-                      verbose=TRUE, silent=FALSE,
+                      pop_temper=max(0.1/seq_alpha, 0.3), verbose=TRUE, silent=FALSE,
```

Comment:
For backwards compatibility, should `pop_temper` default off or is it something that's best to just always be on?

Reply:
It should default off, thanks—this was fixed in tidy SMC but not here.  Next commit will fix here.